src/app/app-routing.module.ts

```diff
@@ -11,10 +11,13 @@ const routes: Routes = [
     path: 'coach',
     loadChildren: () => import('./modules/coach/coach.module').then(m => m.CoachModule),
   },
-  { path: 'coach-categorie', loadChildren: () => import('./modules/coach-type-exercice/coach-type-exercice.module').then(m => m.CoachTypeExerciceModule) },
+  {
+    path: 'athletes',
+    loadChildren: () => import('./modules/athletes/athletes.module').then(m => m.AthletesModule),
```

Comment:
WIP? because this module is missing 🤖 right?

Reply:
yes